### DISCUSSION TITLE
Finish the planning algorithm but manufacturing order still empty

### DISCUSSION THREAD

**Discussion by Drayang** [frePPLe/frepple#481] [Q&A] [answered]
I have execute a constrained plan and the status show me "Done". However, there is no any result show such as manufacturing order and the "Delay" field in sales order. Besides, the planning algorithm take about 20 minute to finish running which in my opinion is not a common case since i only got about 10 sales order to be planned.  The log file didnt show any error to me too. Does anyone have any idea on such issue?
The log file is attached below:
[frepple-20220117060737.log](https://github.com/frePPLe/frepple/files/7881108/frepple-20220117060737.log)

**Comment by jdetaeye** [comment, marked as answer]
20 minutes for 5 orders indicates something is wrong in your model.

Take a sales order and check out what you see in the "why short or late" tab:
![image](https://user-images.githubusercontent.com/5219207/149759677-4f15f599-8c56-4e84-aa90-134e81a314c1.png)

It that doesn't give any conclusive answer, set the parameter "plan.loglevel" to 2 and check the output of the planning run.  You'll get a huge log file...
